src/main/java/spireMapOverhaul/zones/volatileGrounds/monsters/GremlinArchmage.java

```diff
@@ -0,0 +1,183 @@
+import com.badlogic.gdx.math.MathUtils;
+import com.esotericsoftware.spine.AnimationState;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateFastAttackAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateJumpAction;
+import com.megacrit.cardcrawl.actions.common.*;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.status.Burn;
+import com.megacrit.cardcrawl.cards.status.Slimed;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.beyond.Exploder;
+import com.megacrit.cardcrawl.monsters.beyond.SnakeDagger;
+import com.megacrit.cardcrawl.powers.*;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.volatileGrounds.powers.ExplodePlusPower;
+
+import java.util.Iterator;
+
+public class GremlinArchmage extends CustomMonster {
+    public static final String ID = SpireAnniversary6Mod.makeID("GremlinArchmage");
+    private static final MonsterStrings monsterStrings = CardCrawlGame.languagePack.getMonsterStrings(ID);
+    public static final String NAME = monsterStrings.NAME;
+    public static final String[] MOVES = monsterStrings.MOVES;
+    private static final String TEMP_IMG = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/GremlinArchmage.png");
+    //TODO: ADD ANIMATION
+    //private static final String IMG = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/skeleton.png");
+    //private static final String ATLAS = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/skeleton.atlas");
+    //private static final String SKELETON = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/skeleton.json");
+    private static final byte SUMMON = 0;
+    private static final byte ATTACK = 1;
+    private static final byte BLOCK = 2;
+    private static final int ATTACK_DAMAGE_1 = 12;
+    private static final int A2_ATTACK_DAMAGE_1 = 14;
+    private static final int BLOCK_AMOUNT = 8;
+    private static final int A17_BLOCK_AMOUNT = 12;
+    private static final int HP_MIN = 95;
+    private static final int HP_MAX = 98;
+    private static final int A7_HP_MIN = 110;
+    private static final int A7_HP_MAX = 115;
+    private int damage1;
+    private int blockAmt;
+    public AbstractMonster[]  exploders = new AbstractMonster[2];
+    public static final float[] POSX;
+    public boolean firstTurn = true;
+    static
+    {
+        POSX = new float[]{-430, -200.0F};
+    }
+    
+    public GremlinArchmage(final float x, final float y) {
+        super(NAME, ID, HP_MAX, -5.0F, 0, 120.0f, 220f, TEMP_IMG, x, y);
```

Comment:
Something I missed on the first pass -- for this and all the other monsters, you should set `this.type`.

E.g. `this.type = EnemyType.NORMAL;` or `this.type = EnemyType.ELITE;`.